lxc/storage_volume.go

```diff
@@ -218,7 +218,7 @@ func (c *cmdStorageVolumeAttach) run(cmd *cobra.Command, args []string) error {
 	devPath := ""
 	devName := ""
 	if len(args) == 3 {
-		devName = args[1]
+		devName = volName
```

Comment:
I think I disagree with this commit/change; as the `note` in the docs [here](https://documentation.ubuntu.com/lxd/en/latest/howto/storage_volumes/#view-storage-volumes) points out, custom storage volumes and instances can have the same name. After #14532 is merged, `lxc storage volume attach/detach` would generate the same device name for `virtual-machine/vm1` and `custom/vm1`. I don't like introducing that possibility if we can avoid it.

Reply:
You make a fine argument, dropping this commit